test/functional/cmdLineTests/valuetypeddrtests/flattenedvaluetypeddrtests_lw5.xml

```diff
@@ -0,0 +1,233 @@
+		<output regex="no" type="failure">Exception caught!</output>
+	</test>
+
+	<test id="Run !threads on nocompressedrefs core">
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core $DUMPFILE$</arg>
+			<input>!threads</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success" showMatch="yes">!stack 0x</output>
+		<saveoutput regex="no" type="required" saveName="mainThreadId" splitIndex="1" splitBy="!stack ">!stack 0x</saveoutput>
+		<output regex="no" type="failure">DDR is not enabled for this core file</output>
+	</test>
+
+	<test id="Run !stackslots $mainThreadId$ on nocompressedrefs core">
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core $DUMPFILE$</arg>
+			<input>!stackslots $mainThreadId$</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success" showMatch="yes">I-Slot: a0[0x</output>
+		<saveoutput regex="no" type="required" saveName="objectAddr" splitIndex="1" splitBy="= " showMatch="yes">I-Slot: a0[0x</saveoutput>
+		<output regex="no" type="failure">Problem running command</output>
+	</test>
+
+	<test id="Run !j9object $objectAddr$ to display container array contents on nocompressedrefs core">
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core $DUMPFILE$</arg>
+			<input>!j9object $objectAddr$</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success" showMatch="yes">!J9IndexableObject 0x</output>
+		<saveoutput regex="no" type="required" saveName="valueAddr" splitIndex="1" splitBy="= !j9object " showMatch="yes">[0] = !fj9object 0x</saveoutput>
+		<saveoutput regex="no" type="required" saveName="arrayAddr" splitIndex="1" splitBy="= !j9object " showMatch="yes">[2] = !fj9object 0x</saveoutput>
+		<output regex="no" type="failure">Problem running command</output>
+	</test>
+
+	<test id="Run !j9object $valueAddr$ to show the fields are all at 8 bytes aligned offsets on nocompressedrefs core">
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core $DUMPFILE$</arg>
+			<input>!j9object $valueAddr$</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success" showMatch="yes">AssortedValueWithSingleAlignment</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 0)</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 48)</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 56)</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 72)</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 80)</output>
+		<output regex="no" type="required" showMatch="yes">(offset = 88)</output>
+
+		<output regex="no" type="failure">(offset = 4)</output>
+		<output regex="no" type="failure">(offset = 12)</output>
+		<output regex="no" type="failure">(offset = 20)</output>
+		<output regex="no" type="failure">(offset = 28)</output>
+		<output regex="no" type="failure">(offset = 36)</output>
+		<output regex="no" type="failure">(offset = 44)</output>
```

Comment:
The last required condition checks (offset = 88), so I suggest adding failure condition checks upto (offset = 92).